src/main/java/uk/gov/hmcts/reform/sscs/personalisation/SyaAppealCreatedAndReceivedPersonalisation.java

```diff
@@ -28,10 +28,20 @@
         setReasonsForAppealingDetails(personalisation, ccdResponse);
         setHearingDetails(personalisation, ccdResponse);
         setHearingArrangementDetails(personalisation, ccdResponse);
+        setRepresentativeName(personalisation, ccdResponse);
```

Comment:
Interesting. 
In another PR. I got this class to extend WithRepresentativePersonalisation rather than duplicate the code.